### DISCUSSION TITLE
HA-AddOn benötigt einen 2. Neustart, um HMIP-RFUSB zu erkennen.

### DISCUSSION THREAD

**Discussion by tzdisco4** [OpenCCU/OpenCCU#2957] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Wenn ich Raspberrymatic über Systemsteuerung->Zentralen-Wartung herunterfahre und dann über Home Assistant das AddOn neu starte, fehlt auf der Startseite rechts der CCU Status mit Duty Cycle und Carrier sense vollständig.

Vorher hatte ich per Terminal auf dem HA-OS per lsusb geschaut, ob der RFUSB-Stick gelistet wird.

Löse ich dann einen NEustart über das HA-AddOn aus, wird der RFUSB-Stick offensichtlich vollständig erkannt.

Schon beim ersten Start taucht im Log des AddOn folgendes auf:
```
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.3, OK
Identifying Homematic RF-Hardware: ....HmRF: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:00:0c.0-2, HmIP: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:00:0c.0-2, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: 4.4.18, not necessary, OK
```
Gemäß dieser Meldung sollte es schon beim ersten Start funktionieren.

### Describe the behavior you expected

Vollständiges Erkennen des HMIP-RFUSB schon beim ersten start des AddOns unter HA.

### Steps to reproduce the issue

1. Herunterfahren der Rasberrymatic per Systemsteuerung->Zentralen-Wartung
2. Starten des AddOns über HA -> Dastellung CCU-Status fehlt
3. Neustart des AddOns über HA -> Darstellung CCU-Status wieder vorhanden
4.
...
[de838cd8_raspberrymatic_2024-12-16T10-04-02.481Z.log](https://github.com/user-attachments/files/18148739/de838cd8_raspberrymatic_2024-12-16T10-04-02.481Z.log)

### What is the version this bug report is based on?

3.79.6.20241122

### Which base platform are you running?

ha-addon (HomeAssistant Add-on)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
Ich bin auf dies Verhalten eher zufällig gestoßen, nachdem ich wg. Empfangsproblemen mehrfach die Position des Sticks geändert habe und dabei der Sick auch mal den "Anschluss" verloren hat.

Das war dann heute vormittag aauch der Grund für das Herunterfahren.

HA läuft übrigens bei mir in einer Virtualbox auf einem Intel-Nuc I5 mit 8 Gb und nVme-SSD. Host ist Ubuntu 24.04 LTS.
```

### Additional information

[de838cd8-raspberrymatic-2024-12-16-ccu.log](https://github.com/user-attachments/files/18148820/de838cd8-raspberrymatic-2024-12-16-ccu.log)

**Comment by jens-maus** [comment, marked as answer]
> Schon beim ersten Start taucht im Log des AddOn folgendes auf:
> 
> ```
> Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.3, OK
> Identifying Homematic RF-Hardware: ....HmRF: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:00:0c.0-2, HmIP: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:00:0c.0-2, OK
> Updating Homematic RF-Hardware: HMIP-RFUSB: 4.4.18, not necessary, OK
> ```
> 
> Gemäß dieser Meldung sollte es schon beim ersten Start funktionieren.

Tut es doch. Einfach mal ein bisschen mehr geduld haben mit dem auftauchen der DutyCycle und CarrierSense anzeige. Diese taucht nach einem frischen Neustart nicht sofort und unmittelbar auf, sondern es braucht seine zeit. Insofern, "not a bug, but a feature"...

**Comment by tzdisco4** [comment]
Danke sehr für die schnelle Reaktion! Auf "etwas warten" wäre ich nicht gekommen, da meine bisherige, mehrjährige Erfahrung auf Alex Reinerts Debmatic beruht. Dort gibt es zwar keinen Carrier-Sense, aber der DC war immer sofort sichtbar, nachdem neu gestartet wurde. Wieder was dazu gelernt ...
